templates/store/snap-details.html

```diff
@@ -148,6 +148,23 @@ <h1 class="p-heading--two p-snap-heading__name">{{ snap_title }}</h1>
     </div>
   </div>
 
+  <div data-js="blog-articles" style="display: none;">
+    <div class="row">
+      <div class="col-12">
+        <hr />
+      </div>
+    </div>
+    <div class="p-strip is-shallow">
+      <div class="row">
+        <h4>Related blog posts</h4>
+      </div>
+      <div class="row u-equal-height" data-js="blog-article-list" data-snap="{{ package_name }}">
+      </div>
+    </div>
+  </div>
+  {% set article_title = {'rendered': '${title}'} %}
```

Comment:
Is `article_title` used anywhere on it's own? Can't it be defined inside of `article` below?

Reply:
Maybe we could prepare this data in the view then? Both solutions are not perfect, I don't have a strong preference.

Reply:
I thought we might want to use the partial for something else in the future so wanted to keep the variable definitions separate.